sdcm/fill_db_data.py

```diff
@@ -3137,16 +3149,27 @@ def truncate_table(self, session, truncate):  # pylint: disable=no-self-use
         LOGGER.debug(truncate)
         session.execute(truncate)
 
+    @contextlib.contextmanager
+    def _execute_and_log(self, message):
```

Comment:
The usage of context manager is nice, but is it really needed here?
Wouldn't just wrapping the calls with a "self.log.info"  and "timeit" been easier?